_data/current.yml

```diff
@@ -365,6 +343,12 @@
   url: https://waset.org/software-system-testing-and-quality-assurance-conference-in-april-2020-in-new-york?utm_source=testingconferences
   status: CFP is Open until December 30, 2020
 
+- name: SQA Days - 28
+  location: St. Petersburg, Russia
+  dates: "May 21-22, 2021"
+  url: https://sqadays.com/en/index?eventId=82473?utm_source=testingconferences
+  status: Registration is Open, CFP is Open
```

Comment:
SQA Days 27 has registration open. I see nothing about SQA Days 28, do you?

Reply:
thanks, fixed